docs/techniques/healthchecks.md

```diff
@@ -0,0 +1,26 @@
+# Health checks in VirtoCommerce platform
+VirtoCommerce platform support health checks from ASP.NET Core middlewares.
+## Adding heal checks in modules
+### 1. Creating health check
+Create class wich inherits from `IHealthCheck` interface. Implement it.
+![image](../media/health-checks-creating.jpg)
+### 2. Registration
+Register created health checks in service collection.
+1. Open `Module.cs`;
```

Comment:
the text can be replaced with a code snippet, it will be better understand and observe see as an example
https://github.com/VirtoCommerce/vc-platform/blob/master/docs/fundamentals/extensibility/extending-authorization-policies.md#extending-existing-authorization-policies